### ISSUE TITLE
Precompile error: test files do not exist in repo

### ISSUE THREAD

**Issue by tddrmllr** [alchapone/polymer-elements-rails#5]
Getting this error when precompiling assets:

```
can't convert nil into String
  (in /Users/toddmiller/.rbenv/versions/1.9.3-p484/lib/ruby/gems/1.9.1/gems/polymer-elements-rails-1.0.0.pre.rc.1/app/assets/components/gold-cc-cvc-input/test/basic.html)
```

After doing some digging, I found that the tests contain references to the following files, which do not exist in the repo: 
`../../web-component-tester/browser.js`
`../../test-fixture/test-fixture-mocha.js`
`../../iron-test-helpers/test-helpers.js`
`../../test-fixture/test-fixture.html`

Suggestions for how to fix this?

**Comment by tddrmllr** [comment]
Hmm... if I use this in my gemfile `gem 'polymer-elements-rails', github: 'alchapone/polymer-elements-rails'` instead of this `gem 'polymer-elements-rails'`, none of those tests are there and it seems to work fine...

**Comment by alchaplinsky** [comment]
This should now be fixed in v1.0.0
